protocol/src/main/java/org/opensearch/sql/protocol/response/format/Format.java

```diff
@@ -17,7 +17,12 @@ public enum Format {
   JDBC("jdbc"),
   CSV("csv"),
   RAW("raw"),
-  VIZ("viz");
+  VIZ("viz"),
+  // format of explain response
+  SIMPLE("simple"),
+  STANDARD("standard"),
+  CODEGEN("codegen");
```

Comment:
Explain response format is always JSON?

Reply:
Using `format` is not particularly appropriate, mainly because I want to avoid adding an extra parameter.
`_ppl/_explain?format=simple`
Do you think should we add a new parameter for it?

Reply:
Not sure if I understand this right. In Calcite, explain format is actually referring to JSON, XML, DOT? Ref: https://calcite.apache.org/docs/reference.html. If so, I feel making different explain content as different response format is a little confusing.

Reply:
@dai-chen You are refering to Calcite SQL? IMO, it is out of scope of this PR.
@LantaoJin  We should consider to add explain command in PPL in future.